### PR TITLE
docs: Add a sample to demonstrate the evaluation results

### PR DESCRIPTION
**Pull request by DevStephanie**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/snippets/bqml_getting_started_test.py [hunk 1]
```diff
@@ -91,3 +91,50 @@ def test_bqml_getting_started(random_model_id):
...
+    features = bpd.DataFrame(
+        {
+            "os": operatingSystem,
+            "is_mobile": isMobile,
+            "country": country,
+            "pageviews": pageviews,
+        }
+    )
+
+    # Some models include a convenient .score(X, y) method for evaluation with a preset accuracy metric:
```
Comment (tswast) [inline samples/snippets/bqml_getting_started_test.py:135]: Let's also mention that the results are in the same form a `ML.EVALUATE` here. This from the SQL description would be really important to include:

> 
> Because you performed a logistic regression, the results include the following columns:
> 
> [precision](https://developers.google.com/machine-learning/glossary/#precision) — A metric for classification models. Precision identifies the frequency with which a model was correct when predicting the positive class.
> [recall](https://developers.google.com/machine-learning/glossary/#recall) — A metric for classification models that answers the following question: Out of all the possible positive labels, how many did the model correctly identify?
> [accuracy](https://developers.google.com/machine-learning/glossary/#accuracy) — Accuracy is the fraction of predictions that a classification model got right.
> [f1_score](https://en.wikipedia.org/wiki/F1_score) — A measure of the accuracy of the model. The f1 score is the harmonic average of the precision and recall. An f1 score's best value is 1. The worst value is 0.
> [log_loss](https://en.wikipedia.org/wiki/Cross_entropy#Cross-entropy_error_function_and_logistic_regression) — The loss function used in a logistic regression. This is the measure of how far the model's predictions are from the correct labels.
> [roc_auc](https://developers.google.com/machine-learning/glossary/#AUC) — The area under the [ROC](https://developers.google.com/machine-learning/glossary/#ROC) curve. This is the probability that a classifier is more confident that a randomly chosen positive example is actually positive than that a randomly chosen negative example is positive. For more information, see [Classification](https://developers.google.com/machine-learning/crash-course/classification/video-lecture) in the Machine Learning Crash Course.
> 

https://cloud.google.com/bigquery/docs/create-machine-learning-model#evaluate_your_model

Comment (DevStephanie) [reply samples/snippets/bqml_getting_started_test.py:135]: Yes, I agree. Will have those edits today.

File: samples/snippets/bqml_getting_started_test.py [hunk 2]
```diff
@@ -91,3 +91,67 @@ def test_bqml_getting_started(random_model_id):
...
+    # Out of all the possible positive labels, how many did the model correctly identify?
+    # accuracy — Accuracy is the fraction of predictions that a classification model got right.
+    # f1_score — A measure of the accuracy of the model. The f1 score is the harmonic average of
+    # the precision and recall. An f1 score's best value is 1. The worst value is 0.
+    # log_loss — The loss function used in a logistic regression. This is the measure of how far the
+    # model's predictions are from the correct labels.
+    # roc_auc — The area under the ROC curve. This is the probability that a classifier is more confident that
+    # a randomly chosen positive example
+    # is actually positive than that a randomly chosen negative example is positive. For more information,
+    # see Classification in the Machine Learning Crash Course.
```
Comment (tswast) [inline samples/snippets/bqml_getting_started_test.py:151]: Include the link to this course.

Comment (tswast) [inline samples/snippets/bqml_getting_started_test.py:151]: The formatting is a bit funky / hard to read. Please add some bullet points and indentation:

```suggestion
    # * precision -- A metric for classification models. Precision identifies the frequency
    #   with which a model was correct when predicting the positive class.
    # * recall -- A metric for classification models that answers the following question:
    #   Out of all the possible positive labels, how many did the model correctly identify?
    # * accuracy -- Accuracy is the fraction of predictions that a classification model got right.
    # * f1_score -- A measure of the accuracy of the model. The f1 score is the harmonic average of
    #   the precision and recall. An f1 score's best value is 1. The worst value is 0.
    # * log_loss -- The loss function used in a logistic regression. This is the measure of how far the
    #   model's predictions are from the correct labels.
    # * roc_auc -- The area under the ROC curve. This is the probability that a classifier is more
    #   confident that a randomly chosen positive example is actually positive than that a randomly
    #   chosen negative example is positive. For more information, see "Classification" in the
    #   Machine Learning Crash Course at
    #   https://developers.google.com/machine-learning/crash-course/classification/video-lecture
```

Comment (DevStephanie) [reply samples/snippets/bqml_getting_started_test.py:151]: Yes, will do!

Comment (DevStephanie) [reply samples/snippets/bqml_getting_started_test.py:151]: Yes, that does make sense. I will do that now.

**Review by tswast — APPROVED** [review]
Thanks!

File: samples/snippets/bqml_getting_started_test.py [hunk 3]
```diff
@@ -91,3 +91,78 @@ def test_bqml_getting_started(random_model_id):
         replace=True,
     )
     # [END bigquery_dataframes_bqml_getting_started_tutorial]
+
+    # [START bigquery_dataframes_bqml_getting_started_tutorial_evaluate]
+    import bigframes.pandas as bpd
+
+    # Select model you'll use for training. 'read_gbq' accepts either a SQL query
+    # or a table ID.
```
Comment (tswast) [inline samples/snippets/bqml_getting_started_test.py:99]:
```suggestion
    # Select model you'll use for training. `read_gbq_model` loads model data from a
    # BigQuery, but you could also use the `model` object from the previous steps.
```
